docs/UserGuide.md

```diff
@@ -494,15 +494,19 @@ Finds all students with a particular tag.
 Formats: `find t/tagName`
 
 - Only one tag can be searched at each time.
+- The tag is case-sensitive.
```

Comment:
Good catch for including this specification for the behaviour and updating the test case accordingly!